scipy/stats/tests/test_morestats.py

```diff
@@ -2041,6 +2046,22 @@ def test_instability_gh20021(self, xp):
         # expect float64 output for integer input
         xp_assert_close(llf, xp.asarray(-15.32401272869016598, dtype=xp.float64))
 
+    def test_axis(self, xp):
+        data = xp.asarray([[100, 200], [300, 400]])
+        llf_axis_0 = stats.boxcox_llf(1, data, axis=0)
+        data_axes_swapped = xp.moveaxis(data, 0, -1)
+        llf_0 = xp.asarray([
+                stats.boxcox_llf(1, data_axes_swapped[0, :]),
+                stats.boxcox_llf(1, data_axes_swapped[1, :]),
```

Comment:
If you don't mind, I will probably do this in a follow-up.
```suggestion
        llf_0 = xp.asarray([
                stats.boxcox_llf(1, data[:, 0]),
                stats.boxcox_llf(1, data[:, 1]),
```
Although this is really just for non-NumPy backends now. The `test_axis_nan_policy.py` checks `axis` behavior extensively now.